woocommerce/woocommerce
# Use fallback colors wherever CSS variables are used in WooCommerce admin

### Changes proposed in this Pull Request:





There are cases where some WooCommerce admin elements are used in non-WooCommerce pages. For example, select2 elements where have been already fixed by

- https://github.com/woocommerce/woocommerce/pull/40586

When I fixed the default colors for Select2, I missed some elements, which this PR fixes.

Closes #41774  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. With WC 8.3 installed visit `WooCommerce > Settings > Payments`
2. Using the inspector, deactivate all `--wp-admin-theme-color` variables
3. Observe that toggles are displayed like below and not clearly visible

![image](https://github.com/woocommerce/woocommerce/assets/2484390/dd4ef259-cd1b-4120-b71c-52999d74038f)

1. Checkout this branch and build WooCommerce (here's a zip [woocommerce.zip](https://github.com/woocommerce/woocommerce/files/13511566/woocommerce.zip) )
2. Visit `WooCommerce > Settings > Payments`
3. Using the inspector, deactivate all `--wp-admin-theme-color` variables
4. Observe that toggles now have a default color 

![image](https://github.com/woocommerce/woocommerce/assets/2484390/a74ff1aa-fa5f-4735-89f0-7d6135369128)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Use fallback colors when CSS variables are not available in admin.

#### Comment 

</details>
